Documentation/design-docs/raw-eventlistener.md

```diff
@@ -0,0 +1,81 @@
+# Raw EventListener API #
+
+The goal of this design document is to describe what a Raw EventListener API will look like and how it will work.
+
+A raw API is one that dispatches events, but does not decode the data payload.  It simply provides a raw blob and a metadata blob that can be used to interpret the data when desired.
```

Comment:
I have made a document that describes the EventPipe event Serialization format at https://github.com/Microsoft/perfview/blob/master/src/TraceEvent/EventPipe/EventSerialization.md.  You probably want to link to it.